CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -506,6 +506,21 @@ private static void OnIsBrowserInitializedChanged(DependencyObject d, Dependency
 
         protected virtual void OnIsBrowserInitializedChanged(bool oldValue, bool newValue)
         {
+            managedCefBrowserAdapter.GetZoomLevelAsync()
+                .ContinueWith(new Action<Task<double>>((Task<double> previous) =>
+                    {
+                        if (previous.IsCompleted)
+                        {
+                            UiThreadRunAsync(() =>
+                            {
+                                SetCurrentValue(ZoomLevelProperty, previous.Result);
```

Comment:
Is this new code block required? As it's only called once, would the value be anything other than `0`?